cardano-wallet.cabal

```diff
@@ -38,19 +38,30 @@ library
     , bytestring
     , cborg
     , containers
+    , cryptonite
     , deepseq
+    , http-api-data
+    , http-media
+    , memory
+    , servant
+    , servant-client
+    , servant-server
     , text
     , transformers
   hs-source-dirs:
       src
   exposed-modules:
-      Cardano.Wallet.Binary
+      Cardano.HttpBridge.Api
+    , Cardano.HttpBridge.Client
+    , Cardano.Network.Name
+    , Cardano.Wallet.Binary
     , Cardano.Wallet.Binary.Helpers
     , Cardano.Wallet.Binary.Packfile
     , Cardano.Wallet.Primitive
   other-modules:
-      Paths_cardano_wallet
-
+      Codec.CBOR.Extra
+    , Paths_cardano_wallet
+    , Servant.Extra.ContentTypes
```

Comment:
What's the rational for not exposing those :thinking: ?

Reply:
My initial rationale was that these are just utility functions, not part of the library interface that we expose. But I don't feel strongly about it either way. I've unhidden them.